### DISCUSSION TITLE
Grief Prevention doesn't let me interact NPC Villager...

### DISCUSSION THREAD

**Discussion by veploxmine123** [GriefPrevention/GriefPrevention#1498] [Q&A]
It doesn't let me interact the villager npc from cetizens any ideas?

**Comment by bigpresh** [comment]
No, because you've provided pretty much zero useful information - e.g. what version you're using, what you expected should happen, what actually happens, what message (if any) you get, etc.

Please [raise a ticket](https://github.com/TechFortress/GriefPrevention/issues/new/choose) and provide the details asked for in the issue template.

If I had to guess, though - I believe GP requires you to have containertrust in order to interact with villagers within a claim.

**Reply by Jikoo** [reply]
There's a config node `GriefPrevention.Claims.VillagerTradingRequiresPermission` for that, but yeah, without more information it's hard to say.
